### PR TITLE
add victoriacheng15.json

### PR DESCRIPTION
**Pull request by victoriacheng15**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1164"></a>

### CODE REVIEW

**Review by ssensalo — APPROVED** [review]
Looks good

**Review by kumarsonsoff3 — APPROVED** [review]

File: public/data/victoriacheng15.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      "url": "https://victoriacheng15.hashnode.dev/",
+      "icon": "hashnode"
+    },
+    {
+      "name": "Follow me on Polywork",
+      "url": "https://www.polywork.com/victoriacheng15",
+      "icon": "polywork"
+    }
+  ]
+}
```
Comment (Panquesito7) [inline public/data/victoriacheng15.json:33]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/victoriacheng15.json (hunk 1 above)
Comment (krshdev) [reply public/data/victoriacheng15.json:33]: Hey 👋 The bot will fix this 😄

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Review by Pradumnasaraf — APPROVED** [review]
LGTM! Thank you for your valuable contribution.

**Review by krshdev — APPROVED** [review]
Thank you for the contributiong 👍

**Review by emmalearnscode — APPROVED** [review]

**Pull request merged**